pwa/src/components/applicationCard/ApplicationCard.module.css

```diff
@@ -60,7 +64,9 @@
 }
 .interactionCategory,
```

Comment:
A lot of the css in this file isn't used anymore:
`.layerTags` and all the layer colors